### DISCUSSION TITLE
Maybe a future compile error?

### DISCUSSION THREAD

**Discussion by FalloutZone** [juhaku/utoipa#466] [General]
I am extremely new to Rust and leaning while building a little webservice.  I was unable to compile utiopa-gen but fixed it by following the compiler hints.  Not sure what I did, but I assume it's probably a language version issue.

```
utoipa-gen/src/component/schema.rs: Line 54:
    from: const TO_SCHEMA_LIFETIME: &str = "'__s";
    to:   const TO_SCHEMA_LIFETIME: &'a str = "'__s";
utoipa-gen/src/path/response/derive.rs: Line 34:
    from: const LIFETIME: &str = "'__r";
    to:   const LIFETIME: &'r str = "'__r";
```

rustc 1.63.0 (4b91a6ea7 2022-08-08)
Windows 10 64

**Comment by juhaku** [comment]
Oh thanks for letting me know. This is caused by my mistake. I introduced syntax that is available from rust 1.64 onwards and you are using Rust 1.63 what is already quite old. I'll make another release to fix it so it will work again in Rust 1.60.

**Reply by juhaku** [reply]
https://github.com/juhaku/utoipa/pull/467 Here is the PR for this I will make another release soon what will fix this issue.

**Reply by juhaku** [reply]
With this latest release 3.0.1 https://github.com/juhaku/utoipa/releases/tag/utoipa-3.0.1 things should work ok.

**Comment by FalloutZone** [comment]
awesome! thx
